tensorflow_cloud/run.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import sys
+
+from . import containerize
+from . import deploy
+from . import gcp
+from . import machine_config
+from . import package
+from . import preprocess
+from . import validate
+
+
+# Flag which indicates whether current process is running in a cloud
+# environment created by the `cloud.run` API.
+_IS_RUNNING_REMOTELY = False
+
+
+def _is_running_remotely():
+    return _IS_RUNNING_REMOTELY
+
+
+def _set_running_remotely(value):
+    global _IS_RUNNING_REMOTELY
+    _IS_RUNNING_REMOTELY = value
+
+
+def run(
+    entry_point,
+    requirements_txt=None,
+    distribution_strategy='auto',
+    docker_base_image=None,
+    chief_config='auto',
+    worker_config='auto',
+    worker_count=0,
+    region=None,
+    entry_point_args=None,
+    stream_logs=False,
+):
+    """Runs your Tensorflow code in Google Cloud Platform.
+
+    # Arguments:
```

Comment:
You use sometimes `Args:`, and here, `# Arguments:`. I guess we should prefer `Args:`; in any case we should be consistent.

Reply:
Fixed, using `Args:` throughtout.